Runtime/Autopilot.cs

```diff
@@ -134,10 +114,11 @@ public async UniTask TerminateAsync(ExitCode exitCode, string logString = null,
 
             Destroy(this.gameObject);
 
-            if (_state.launchFrom == AutopilotState.LaunchType.EditorPlayMode)
+            if (_state.IsLaunchFromPlayMode)
             {
                 _logger.Log("Terminate autopilot");
-                _state.Reset();
+                _state.settings = null;
+                _state.exitCode = exitCode;
                 return; // Only terminate autopilot run if starting from play mode.
             }
```

Comment:
@nowsprinting Sorry for my late reply. Why `launchFrom` does not get reset in here? I think this is a breaking change.

Reply:
Thanks! Fixed with a7f2c65

Fix regression about AutopilotState after terminate

- Revert to "reset AutopilotState in TerminateAsync"
- Changed throw NUnit.Framework.AssertionException to be during in the TerminateAsync